Makefile

```diff
@@ -55,6 +55,7 @@ pkgconfig_DEPS =
 readline_DEPS = ncurses
 ruby_big_DEPS = zlib openssl gmp ncurses readline
 ruby_DEPS = zlib openssl ncurses readline
+xz_DEPS = glib
```

Comment:
I think you are right. It is not depending on glib.

I copied from the other line when I was trying to make it pick up xz, along with adding the targets. I think the targets are the only thing needed, not the dependency.

Thanks.